src/sentry/static/sentry/app/views/performance/realUserMonitoring/content.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {Location} from 'history';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import omit from 'lodash/omit';
+
+import CreateAlertButton from 'app/components/createAlertButton';
+import * as Layout from 'app/components/layouts/thirds';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {decodeScalar} from 'app/utils/queryString';
+import SearchBar from 'app/views/events/searchBar';
+
+import TransactionVitals from './transactionVitals';
+import TransactionHeader, {Tab} from '../transactionSummary/header';
+
+type Props = {
+  location: Location;
+  eventView: EventView;
+  transactionName: string;
+  organization: Organization;
+  projects: Project[];
+};
+
+type State = {
+  incompatibleAlertNotice: React.ReactNode;
+};
+
+class RumContent extends React.Component<Props, State> {
+  state: State = {
+    incompatibleAlertNotice: null,
+  };
+
+  handleSearch = (query: string) => {
+    const {location} = this.props;
+
+    const queryParams = getParams({
+      ...(location.query || {}),
+      query,
+    });
+
+    // do not propagate pagination when making a new search
+    const searchQueryParams = omit(queryParams, 'cursor');
```

Comment:
You can also do `queryParams.cursor = undefined`

Reply:
I realized I can just do `delete queryParams.cursor` here and remove the dependency on `lodash`.

Reply:
Addressed in 33a7282